src/app/accounts/destiny-account.ts

```diff
@@ -168,9 +164,9 @@ async function generatePlatforms(
           originalPlatformType: destinyAccount.membershipType,
           membershipId: destinyAccount.membershipId,
           platformLabel: PLATFORM_LABELS[destinyAccount.membershipType],
-          destinyVersion: 1,
```

Comment:
This was a bug - errored D2 accounts would appear as D1 accounts. Adding the test helped me find this.